(Fix) Add details about artifact security (#6000)
Kong/docs.konghq.com: app/_src/gateway/production/security-update-process.md

---
title: Kong Security Update Process
---

> ## Security of Release Artifacts

Author: ```suggestion

```

> Kong provides artifacts in multiple formats, such as deb, rpm, apk, that are free of vulnerabitilies at the time of release. Occasionally, for convenience, Kong also provides Docker images that are also free of vulnerabilities at the time of release.

Reviewer: ```suggestion
Kong provides artifacts in [multiple formats](/gateway/{{page.kong_version}}/support-policy/#supported-versions), such as deb, rpm, apk, that are free of vulnerabilities at the time of release. Occasionally, for convenience, Kong also provides Docker images that are also free of vulnerabilities at the time of release.
```

Author: added as separate edit on [e613689](https://github.com/Kong/docs.konghq.com/pull/6000/commits/e6136894b23733593d6428760a352707fa5e279d)

> ## Types of Vulnerabililites

Author: ```suggestion
## Types of Vulnerabilities
```

> Generally, there are three types of vulerabilities:

Reviewer: ```suggestion
Generally, there are three types of vulnerabilities:
```

* In Kong code
* In third-party code that Kong directly links with

> * In third-pary codde that is part of the Docker image.

Reviewer: ```suggestion
* In third-pary codde that is part of the Docker image
```

> * In third-pary codde that is part of the Docker image.

Author: ```suggestion
* In third-pary code that is part of the Docker image.
```

> If vulnerabilities are found in Kong code or in third-party code that Kong directly links with, these are addressed within an SLA based on severity and release as patches. 

Reviewer: ```suggestion
If vulnerabilities are found in Kong code or in third-party code that Kong directly links with, these are addressed within an SLA based on severity and released as patches. 
```

If vulnerabilities are found in the third-party code that is part of the Docker image, then customers have the option to: 

> * upgrade to the latest patch Docker image for their major version
> * use the Kong artifacts to build their own Docker image on their corporate-approved Docker distros, using their approved security processes. This is the recommended option. 

Reviewer: ```suggestion
* Use the Kong artifacts to build their own Docker image on their corporate-approved Docker distributions, using their approved security processes. This is the recommended option. 
* Upgrade to the latest patch Docker image for their major version.
```
I'd put the recommended option first (+ some typo fixes)
